doc/nrf-bm/libraries/timer.rst

```diff
@@ -1,15 +1,18 @@
 .. _lib_bm_timer:
 
-Timer library
-#############
+BM Timer library
```

Comment:
This looks a bit ugly in my opinion. I would follow the wording used in the `bm_zms` library and such and write instead the following:
```suggestion
Bare Metal Timer library
```

At the bare minimum we need to align everything to the same style, one way or the other.